.dockerignore

```diff
@@ -1,3 +1,3 @@
 # exclude all files and folders except bin folder
-**
+#**
```

Comment:
Why?

Reply:
need to add project code to docker builder

in dockerfile
```
FROM golang:1.13 as builder
ADD . /opt/kubesphere/
```

Reply:
This change is conflicts with the normal docker build. Please see [line-1](https://github.com/kubesphere/kubesphere/pull/3704/files#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5R1)

In order to avoid making the context too big. We need to exclude some directories. For example: `.git`

Reply:
agree with you. The main thing of the PR is to describe the idea clearly first. I'll change this file